server/scsynth/SC_Lib_Cintf.h

```diff
@@ -43,7 +44,7 @@ extern Malloc gMalloc;
 extern HashTable<class SC_LibCmd, Malloc> *gCmdLib;
 
 void initialize_library(const char *mUGensPluginPath);
-void deinitialize_library();
+SCSYNTH_DLLEXPORT_C void deinitialize_library();
```

Comment:
scsynth internal server is initialized with World_New but there is a flag to initialize some things only the first time it is called (plugins loading is one of them). Then World_CleanUp frees some resources but not those initialized with the flag so that World_New and World_CleanUp can be called several times. (only happens in internal server)

If I unload plugins in World_CleanUp I must also let them initialize each time World_New is called (despite of the init-once flag) and I feel this is too intrusive in the previous working of scsynth, so that a lot of new crashes could appear caused by multiple initialization and deinitialization of other elements in scsynth (plugins included).